src/frontend/Komponenter/Behandling/Modal/HenleggModal.tsx

```diff
@@ -34,9 +46,28 @@ export const HenleggModal: FC<{ behandling: Behandling }> = ({ behandling }) =>
         nullstillIkkePersisterteKomponenter,
         settIkkePersistertKomponent,
     } = useApp();
+    const { vergemål, fullmakt } = personopplysninger;
     const [henlagtårsak, settHenlagtårsak] = useState<EHenlagtårsak>();
+    const [huketAvSendBrev, settHuketAvSendBrev] = useState<boolean>(true);
     const [låsKnapp, settLåsKnapp] = useState<boolean>(false);
     const [feilmelding, settFeilmelding] = useState<string>();
+    const [forhåndsvisningsFeil, settForhåndsvisningsFeil] = useState<string>();
+
+    const visBrevINyFane = () => {
+        axiosRequest<string, null>({
```

Comment:
Bør kanskje ha inn `settForhåndsvisningsFeil('')` før vi kjører kallet - for å nullstille potensielle feil fra tidligere

Reply:
I tillegg, burde man få lov til å gjøre flere requests etter hverandre, eller burde man ha en sjekk på om man allerede har en pågående request? Det er generelt god praksis